scripts/production/Verify_Compara_REST_Endpoints.pl

```diff
@@ -501,9 +531,10 @@ sub verify_xml_leaf {
         print $server.$ext.';content-type=text/x-phyloxml;aligned=0' . "\n";
         $phyloXml = process_phyloXml_get($server.$ext.';content-type=text/x-phyloxml;aligned=0'.($extra_params ? ";$extra_params" : ''));
         #print Dumper $phyloXml;
-        ok($phyloXml->{phylogeny}->{clade}->{sequence}->{mol_seq}->{is_aligned} == 0, "Check get alignment region and unaligned sequences");
+        my $species_node = fetch_species_node($phyloXml->{phylogeny}, $species_1);
+        ok($species_node->{sequence}->{mol_seq}->{is_aligned} == 0, "Check get alignment region and unaligned sequences");
     
-        $jsontxt = process_json_get($server."/alignment/region/$species_1/$lastz_alignment_region?content-type=application/json;display_species_set=$species_1".($extra_params ? ";$extra_params" : ''));
+        $jsontxt = process_json_get($server."/alignment/region/$species_1/$lastz_alignment_region?content-type=application/json;display_species_set=$species_1;species_set_group=$species_set_group".($extra_params ? ";$extra_params" : ''));
```

Comment:
Can you check it still works for Vertebrates ? For Vertebrates the endpoint used to query the mammals alignment (default species set), but now it is querying the primates one

Reply:
I did the test on both vertebrates and plants to make sure I did not introduced more regression before pushing and it did work